README.md

```diff
@@ -134,9 +129,10 @@ The general command to deploy this to an OpenShift cluster is:
 
 ```bash
 helm template <RELEASE> ./devops/charts/controller
+-f ./devops/charts/controller/values_<ENVIRONMENT>.yaml
 --set-string tenant_id=<TENANT_ID> \
 --set-string tenant_api_key=<TENANT_API_KEY> \
---set-string redis_url=<REDIS_URL> \
+--set-string tenant_legacy_did=<TENANT_LEGACY_DID> \
```

Comment:
I just noticed I used `TRACTION_LEGACY_DID` in my PR that was merged this morning but here you have `TENANT_LEGACY_DID`. Which one is correct?